src/operators/expression.md

```diff
@@ -0,0 +1,121 @@
+# Raster Expression
+
+The `Expression` operator performs pixel-wise expression on one or more raster sources.
```

Comment:
suggestion: The `Expression` operator performs *A* pixel-wise *MATHEMATICAL* expression on one or more raster sources.

Reply:
53b8f89